.github/actions/setup-environment/action.yml

```diff
@@ -10,6 +10,8 @@ inputs:
     type: bool
   eve_image:
     type: string
+  eve_artifact_name:
```

Comment:
Should we pass the input through test and run-eden-test actions?

Reply:
Updated, nice catch @giggsoff